src/app/shared/model/series.model.ts

```diff
@@ -19,8 +19,9 @@ export class SeriesModel extends BaseModel implements HasUpload {
   public images: ImageModel[] = [];
   public versionTemplates: AudioVersionTemplateModel[] = [];
   public distributions: DistributionModel[] = [];
+  public accountId: number;
 
-  SETABLE = ['title', 'description', 'shortDescription', 'hasUploadMap'];
+  SETABLE = ['title', 'description', 'shortDescription', 'hasUploadMap', 'accountId'];
```

Comment:
accountId field is not persisted, just a place to manage it. I needed a model to stash it on to make all this work together though. Account doesn't quite make sense to me as a related field, but maybe that's how I should have done it.